lib/frame.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+class Frame
+  TEN_PINS = 10
```

Comment:
Better to name this constant NUMBER_OF_PINS, because TEN_PINS is just as inflexible as using the number 10. If I asked you to adapt your program for a version of bowling that had 15 pins instead of 10, for example, this constant name wouldn't make sense any more.